Makefile

```diff
@@ -44,6 +44,14 @@ else
   BUILD_PROJECT := livecode
 endif
 
+# Prettifying output for CI builds
+ifeq ($(TRAVIS),true)
+  XCODEBUILD := set -o pipefail && $(XCODEBUILD)
+  XCODEBUILD_FILTER := | xcpretty
+else
+  XCODEBUILD_FILTER :=
+endif 
```

Comment:
I think it would be better for the build control files in git to have no explicit knowledge of Travis.  For example, I'm very sorely tempted to install <kbd>xcpretty</kbd> on Vulcan...

What about:

```Makefile
XCODEBUILD_FILTER ?=
```

and then let the `.travis.yml` file set the `XCODEBUILD` and `XCODEBUILD_FILTER` variables as appropriate?